lib/Game.js

```diff
@@ -4,22 +4,36 @@ class Game {
     this.guess = '';
     this.instructions = '';
     this.currentRound = 1;
+    this.puzzleBank = {};
   }
 
-  startGame(playerArray) {
-    let player1 = new Player(playerArray[0]);
-    let player2 = new Player(playerArray[1]);
-    let player3 = new Player(playerArray[2]);
-    let round1 = new Round(this.currentRound);
+  getPlayers(playerArray) {
+    this.player1 = new Player(playerArray[0]);
+    this.player2 = new Player(playerArray[1]);
+    this.player3 = new Player(playerArray[2]);
+  }
+  
+  startRound() {
+    let round = new Round(this.currentRound);
+
+  }
+  
+  updateRound() { 
+    this.currentRound++;
+
+    // if a player has solved the puzzle then we want to move on to the next round // and then we want to update our current now to the next round, and get new data for our puzzle and wheel
+  }
+
+  getPuzzleBank() {
+    let puzzle = new Puzzle();
+    this.puzzleBank = puzzle.puzzleBank();
   }
 
 // giveInstructions() {// our element that updates instruction text should indicate which players turn it is and tell them to spin the wheel// 'player1(name) spin the wheel'// after they spin the wheel we want to then want them to pick a letter from our key board// this should enable the buttons on the keyboard//  the buttons on the keyboard should be disabled after each selection// if their selection is correct then they can keep guessing  // }
```

Comment:
Seems like `giveInstructions` should be a method grouped with your other DOM manipulation methods based on its description. `Game` would just pass this method the information that it needs (e.g. Which player's turn it is).

Reply:
That's the plan. I think we have some confusion as to whether the DOM manipulation methods should ONLY exist in the domUpdates file, or if the domUpdates file is only for the purposes of testing. (It seems like many people in class are confused on this point.)